torch_geometric/transforms/to_sparse_tensor.py

```diff
@@ -65,19 +83,30 @@ def __call__(
             for key, value in zip(keys, values):
                 store[key] = value
 
-            store.adj_t = SparseTensor(
-                row=store.edge_index[1], col=store.edge_index[0],
-                value=None if self.attr is None or self.attr not in store else
-                store[self.attr], sparse_sizes=store.size()[::-1],
-                is_sorted=True, trust_data=True)
+            if self.backend == 'torch_sparse':
+                store.adj_t = SparseTensor(
+                    row=store.edge_index[1], col=store.edge_index[0],
+                    value=None if self.attr is None or self.attr not in store
+                    else store[self.attr], sparse_sizes=store.size()[::-1],
+                    is_sorted=True, trust_data=True)
+            else:
+                store.adj_t = to_torch_coo_tensor(
```

Comment:
Should we use `to_torch_csr_tensor` instead?

Reply:
Do we want to let the user choose the layout. The default could be `csr`.